R/readNWISunit.r

```diff
@@ -387,13 +387,19 @@ readNWISmeas <- function (siteNumbers,startDate="",endDate="", tz="", expanded=F
 #' sites <- c("434400121275801", "375907091432201")
 #' data2 <- readNWISgwl(sites, '','')
 #' data3 <- readNWISgwl("420125073193001", '','')
+#' data4 <- readNWISgwl("425957088141001", startDate = "1980-01-01") #handling of data where date has no day
 #' }
 readNWISgwl <- function (siteNumbers,startDate="",endDate="", convertType = TRUE){  
   
   url <- constructNWISURL(siteNumbers,NA,startDate,endDate,"gwlevels",format="tsv")
   data <- importRDB1(url,asDateTime=TRUE, convertType = convertType)
 
   if(nrow(data) > 0){
```

Comment:
Let's add `convertType` when we are doing the conversions, so the if statement would be:

```r
if(nrow(data) >0){
  if(convertType){
     #...all the date conversion lines...
  }
  siteInfo <- #...still want to do this stuff whether or not the user chooses `convertType`
}

```